internal/provider/jwt_issuers_test.go

```diff
@@ -0,0 +1,385 @@
+/*
+ Copyright 2024 The Cockroach Authors
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package provider
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"regexp"
+	"testing"
+
+	"github.com/cockroachdb/cockroach-cloud-sdk-go/v3/pkg/client"
+	mock_client "github.com/cockroachdb/terraform-provider-cockroach/mock"
+	"github.com/golang/mock/gomock"
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/knownvalue"
+	"github.com/hashicorp/terraform-plugin-testing/statecheck"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/hashicorp/terraform-plugin-testing/tfjsonpath"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	testIssuerURL = "https://issuer-%s.com"
+
+	testAudience        = "audience"
+	testAudienceUpdated = "audience_updated"
+
+	testJWKS        = "{}"
+	testJWKSUpdated = "{updated: 1}"
+
+	testClaim        = "sub"
+	testClaimUpdated = "email"
+
+	testTokenID1 = "token_id1"
+	testCCID1    = "cc_id1"
+
+	testTokenID2 = "token_id2"
+	testCCID2    = "cc_id2"
+
+	testIdentityMap = []IdentityMapEntry{
+		{
+			TokenIdentity: types.StringValue(testTokenID1),
+			CcIdentity:    types.StringValue(testCCID1),
+		},
+		{
+			TokenIdentity: types.StringValue(testTokenID2),
+			CcIdentity:    types.StringValue(testCCID2),
+		},
+	}
+	testIdentityMapUpdated = []IdentityMapEntry{
+		{
+			TokenIdentity: types.StringValue(testTokenID1),
+			CcIdentity:    types.StringValue(testCCID1),
+		},
+	}
+)
+
+// TestAccJWTIssuerResource attempts to create, check, and destroy
+// a real JWT Issuer. It will be skipped if TF_ACC isn't set.
+// In order to work, the `JwtIssuerEnabled` Feature Flag must be enabled and
```

Comment:
I just enabled `JwtIssuerEnabled` for the prod terraform acceptance test org.

Reply:
Thank you, but I am surprised how the acceptance tests worked earlier.